lib/Doctrine/Sequence/Db2.php

```diff
@@ -77,10 +77,10 @@ public function nextId($seqName, $ondemand = true)
     public function currId($sequenceName)
     {
         $sql = 'SELECT PREVVAL FOR '
-             . $this->quoteIdentifier($this->conn->formatter->getSequenceName($sequenceName))
+             . $this->conn->quoteIdentifier($this->conn->formatter->getSequenceName($sequenceName))
              . ' AS VAL FROM SYSIBM.SYSDUMMY1';
 
-        $stmt   = $this->query($sql);
+        $stmt   = $this->conn->query($sql);
```

Comment:
All of these methods don't exist on the current object, but do exist on the connection.